src/components/Search.jsx

```diff
@@ -24,11 +25,36 @@ import { getData, getNumberOfCompany } from "../utils/fetchData";
 import { findParamInURL, updateUrlParams } from "../utils/urlManipulation";
 import Button from "./Button";
 
+function tagMapper([key, currentArray]) {
+  return currentArray.map((item) => (
```

Comment:
Let's map over the new Button.jsx ( use buttonType="addFilters" for the same UI effect); 
this is an old structure that was removed(we need many Buttons not many divs with buttons behavior ):
      <h3>{item}</h3>
      {/* Call removeTag with the specific type and value */}
      <button onClick={() => this.removeTag(key, item)}>
        <img src={removeIcon} alt="x" className="cursor-pointer ml-2" />
      </button>
      
      please check the new Buttons:
      // const FilterTags = ({ tags, removeTag }) => {
//   return (
//     <div className="flex gap-2 flex-wrap">
//       {Object.entries(tags).map(([key, currentArray]) =>
//         currentArray.map((item) => (
//           <Button
//             key={item}
//             buttonType="addFilters"
//             onClick={() => removeTag(key, item)}
//           >
//             {item}
//             <img src={removeIcon} alt="x" className="cursor-pointer ml-2" />
//           </Button>
//         ))
//       )}
//     </div>
//   );
// };